wp-content/themes/ipbs-largo/homepages/assets/less/homepage.less

```diff
@@ -1 +1,112 @@
 @import (reference) "../../../less/variables.less";
+@border-color: @white;
+
+#top-story {
+	position: relative;
+}
+.inner {
+	background-color: @darkblue;
+	padding: @measure;
+	article {
+		border-bottom: 1px solid @border-color;
+		padding-bottom: 2 * @measure;
+		margin: 0;
+	}
+	.menu-container {
+		padding-top: 2 * @measure;
+	}
+
+	@media ( min-width: 770px ) {
+		background-color: fadeout( @darkblue, 20% );
+
+		position: absolute;
+
+		bottom: @measure;
+		left: @measure;
+		right: @measure;
+
+		display: flex;
+		flex-wrap: wrap;
+
+		article {
+			border-bottom: none;
+			flex: 1 1 400px;
+			padding-right: 1 * @measure;
+		}
+		.menu-container {
```

Comment:
Should we add `padding-top: 0;` to this? Currently it's inheriting `padding-top: 3rem;` which is putting the menu out of line with the post title

3rem:
![Screen Shot 2020-01-27 at 9 53 13 AM](https://user-images.githubusercontent.com/18353636/73184822-0692d600-40eb-11ea-802c-faf50af5ff1e.png)

0:
![Screen Shot 2020-01-27 at 9 53 36 AM](https://user-images.githubusercontent.com/18353636/73184821-0692d600-40eb-11ea-897f-666f6a05fb35.png)

Reply:
Fixed in 2cb533d:

Desktop:
![Screen Shot 2020-01-27 at 15 31 47](https://user-images.githubusercontent.com/1754187/73211438-3c02e800-411a-11ea-96a6-6053ab303d67.png)

Mobile:
![Screen Shot 2020-01-27 at 15 31 55](https://user-images.githubusercontent.com/1754187/73211439-3c02e800-411a-11ea-8ace-83f09dafbd1d.png)